Samples/hello-world/outlook-hello-world/README.md

```diff
@@ -94,6 +99,44 @@ function sayHello() {
 
 For more information see [Build your first Outlook add-in](https://learn.microsoft.com/office/dev/add-ins/quickstarts/outlook-quickstart)
 
+## Run the Teams manifest version of hello world
+
+If you want to run the hello world sample using the preview Teams manifest (**manifest.json** or **manifest-localhost.json**), follow the steps below. The Teams manifest files perform the exact same function as the XML manifest files. They help Outlook get information about your add-in, and load the correct resources. For more information, see [Teams manifest for Office Add-ins (preview)](https://learn.microsoft.com/office/dev/add-ins/develop/json-manifest-overview).
+
+>Note: The new manifest is available for preview and is subject to change based on feedback. We encourage experienced add-in developers to experiment with it. The preview manifest should not be used in production add-ins.
```

Comment:
```suggestion
>Note: The new manifest is available for preview in Outlook on Windows and is subject to change based on feedback. We encourage experienced add-in developers to experiment with it. The preview manifest should not be used in production add-ins.
```

Reply:
If you accept this change, it may also help to add a similar client support note to the "Applies to" section earlier in the README.